pkg/operator/ceph/cluster/watcher.go

```diff
@@ -516,7 +521,7 @@ func (c *clientCluster) createNetworkFence(ctx context.Context, pv corev1.Persis
 
 	networkFence := &addonsv1alpha1.NetworkFence{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      node.Name,
+			Name:      node.Name + driver,
```

Comment:
How about also add a `-` to the name? It will be more standard K8s convention and also more readable.
```suggestion
			Name:      node.Name + "-" + driver,
```